src/rpdk/core/contract/suite/handler_create.py

```diff
@@ -63,6 +66,7 @@ def _create_with_invalid_model(resource_client):
 
 
 @pytest.mark.create
+@skip_not_writable_identifier
```

Comment:
this marker is only used once? are there more potential uses down the line? as an aside i think we should inline the `test_create_failure_if_repeat_writeable_id` since it's only being called in this method as well for readability

Reply:
I think it could be used more than once but I am not sure